proposals/0000-bundle-with-type-synonym.rst

```diff
@@ -0,0 +1,138 @@
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+
+.. highlight:: haskell
+
+Bundle pattern synonyms with a type synonym
+===========================================
+
+With `AssociatingSynonyms <https://ghc.haskell.org/trac/ghc/wiki/PatternSynonyms/AssociatingSynonyms>`_
```

Comment:
Ah, I misunderstood the remark. Indeed the ghc manual refers to this as *bundling* already.